### ISSUE TITLE
Add give to player

### ISSUE THREAD

**Issue by Misio12320** [SkriptLang/Skript#3015]
Add:
```vb
on event:
    increase player's protection by 5
    add 5 to player's protection
    remove 5 from player's currect protection (should remove 5 from player's protection and make player -5 protection)
```

**Comment by ShaneBeee** [comment]
What exactly are you wanting here?

**Comment by Misio12320** [comment]
Player have oh I think its not possible. Like default player have 1 damage, 10 protection, and 20% penetration damage. So if I remove player's protection - 10 then penetration deal a lot of damage cause player have 0 protection. Hm I think its not possible

**Comment by FranKusmiruk** [comment]
There's a PR for attributes that'll be hopefully merged soon (see #2664).
